### ISSUE TITLE
Why NanaZip doesn't have a language option unlike 7-Zip

### ISSUE THREAD

**Issue by Comeonnoob** [M2Team/NanaZip#465]
I like to use different languages on apps, why do I have to change the first listed language under Windows' settings? 
Please add the language toggle back! And not use the UWP platform's method to auto detect language (because Windows' own language can be different compared to the first listed language in Language settings), and rely on system locale (en-US) instead.
![изображение](https://github.com/user-attachments/assets/7eef5c44-e942-456c-a1aa-b4d68e25d48d)
![изображение](https://github.com/user-attachments/assets/ab43c7fd-0407-4fc8-8b61-8524565baed8)
![изображение](https://github.com/user-attachments/assets/579c92b9-e623-47c7-8116-2dd9879217e4)
![изображение](https://github.com/user-attachments/assets/39c75b81-6160-453f-ae65-af69da7e9f5e)

**Comment by Miami-T** [comment]
Me too. I have all the apps in English, but my native language on which the date, time and currency format is based is different.

**Comment by Tarvoskemwer** [comment]
Agree, just uninstalled and tried preview too, can't select language there either. So I have to go back to 7-zip.

**Comment by XP-online** [comment]
+1

**Comment by ntd252** [comment]
This needs more votes, please!

**Comment by DervishD** [comment]
+1
